pkg/parser/node.go

```
@@ -377,6 +391,15 @@ func (n *Node) compile(pctx *UnixParserCtx) error {
 		if err != nil {
 			return fmt.Errorf("compilation of '%s' failed: %v", n.Filter, err)
 		}
+
+		if n.Debug {
+			visitor := &exprhelpers.Visitor{}
+			n.DebugExprs, err = visitor.Build(n.Filter)
+			if err != nil {
+				log.Errorf("unable to build debug filter for '%s' : %s", n.Filter, err)
```

I think we might want to return an error here, or shouldn't be a failure ?